viewing_party/party.py

```diff
@@ -1,23 +1,147 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {}
 
+    if title and genre and rating:
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating
+        return movie
+    
+    else:
+        return None

```

Comment:
This works well! The one tweak you might want to consider is a common pattern with functions! If we have a situation that would cause the function to stop early, it's best practice to make that a guard clause and include the regular logic of the function outside of the guard clause. This often means checking the negative rather than the positive! For this function, the pattern would be:

```suggestion
    if not title or not genre or not rating:
        return None
        
    movie["title"] = title
    movie["genre"] = genre
    movie["rating"] = rating
    return movie

```